.evergreen/csfle/set-temp-creds.sh

```diff
@@ -0,0 +1,30 @@
+# Obtains temporary AWS credentials for CSFLE testing.
+#
+# Run with a . to add environment variables to the current shell:
+# . ./set-temp-creds.sh
+#
+# Requires the python AWS SDK boto3. This can be installed with: pip install boto3
+# The path to python in a virtual environment may be passed with the PYTHON
+# environment variable.
+#
+# Requires AWS credentials for CSFLE to obtain temporary credentials.
+# Those credentials can be passed through the following environment variables:
+#
+# export AWS_ACCESS_KEY_ID=...
+# export AWS_SECRET_ACCESS_KEY=...
+# export AWS_DEFAULT_REGION=us-east-1
+#
+# After running this script, the following shell variables are set:
+# - CSFLE_AWS_TEMP_ACCESS_KEY_ID
+# - CSFLE_AWS_TEMP_SECRET_ACCESS_KEY
+# - CSFLE_AWS_TEMP_SESSION_TOKEN
+#
+
+set +o xtrace # Disable tracing.
+
+PYTHON=${PYTHON:-python}
+CREDS=$($PYTHON print-temp-creds.py)
```

Comment:
Rather than relying on this script to get invoked in the same directory as the Python script, maybe we could use a heredoc? (see https://unix.stackexchange.com/questions/624875/is-there-a-way-to-use-heredoc-for-bash-and-zsh-and-be-able-to-use-arguments#answer-624892)

```
get_creds() {
    $PYTHON - "$@" << 'EOF'
import boto3

client = boto3.client('sts')
credentials = client.get_session_token()["Credentials"]
print (credentials["AccessKeyId"] + " " + credentials["SecretAccessKey"] + " " + credentials["SessionToken"])
EOF
}

PYTHON=${PYTHON:-python}
CREDS=$(get_creds)
```

It's messy, but it will make it easier for people to use, and honestly, when is bash _not_ messy?

Reply:
Oh thank you for writing that out. One less script does seem like a win.

> when is bash not messy?

Good point!